src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```
@@ -35,10 +39,10 @@ secondary = true
 
 # Main dependencies
 [tool.poetry.dependencies]
-python = ">=3.7, <3.9"
+python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
-ansys-saf-glow = {version = "0.1.9", source = "solutions-private-pypi" }
-ansys-saf-portal = {version = "0.1.4", source = "solutions-private-pypi"}
+ansys-saf-glow = {version = "0.2", source = "solutions-private-pypi" }
```

Shall we use caret notation instead so that solution dev can use  poetry update easily?
```suggestion
ansys-saf-glow = {version = "^0.2", source = "solutions-private-pypi" }
```